mlflow/entities/span.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+import json
+from dataclasses import asdict, dataclass, field
+from typing import Any, Dict, List, Optional, Union
+
+from mlflow.entities._mlflow_object import _MLflowObject
```

Comment:
not a comment - Sad to see this private module + private class is being used more...

Reply:
yea...it's sad saga of the language...🥲